front-end/src/components/MainPage/MainPage.js

```diff
@@ -58,7 +60,23 @@ const MainPage = () => {
       </AppBar>
       <div>
         <h1>HELLO TEAM TRUFFLE</h1>
-        <CenterConsole/>
+        {/* The Grid system below should be able to hold the left tool bar, center console, and
```

Comment:
Awesome setup!

Reply:
Thank Rick, I really appreciate the review.